src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -3600,12 +3599,11 @@ impl Step for TestFloatParse {
             &[],
         );
 
-        cargo_run.arg("--");
-        if builder.config.args().is_empty() {
-            // By default, exclude tests that take longer than ~1m.
-            cargo_run.arg("--skip-huge");
-        } else {
-            cargo_run.args(builder.config.args());
+        if env::var("FLOAT_PARSE_TESTS_SKIP_HUGE").unwrap() == "1" {
```

Comment:
Suggestion: wouldn't this panic if the env var is unset in other CI jobs?

```suggestion
        if env::var("FLOAT_PARSE_TESTS_SKIP_HUGE").is_some() {
```